### ISSUE TITLE
Profile page posts don't have an intuitive interface

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#2196]
**Describe the bug**
- Please work with @Ayush0Chaudhary for comments, advice and review of your PR.

Profile page posts don't have an intuitive interface. The tiling method doesn't match the newsfeed layout which is unexpected.

**To Reproduce**

1. Start app
2. Click on Profile
3. Select posts

**Expected behavior**
- A post layout that better matches that of the newsfeed

**Actual behavior**
- Tiled layout

**Screenshots**
- https://www.youtube.com/watch?v=86AzK_Y_GCQ&t=505s

**Additional details**
- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by ShubhamTiwari55** [comment]
@Ayush0Chaudhary Please assign this issue to me. I want to resolve this issue.

**Comment by ShubhamTiwari55** [comment]
@Ayush0Chaudhary Can I do it this way ?
<img src="https://github.com/PalisadoesFoundation/talawa/assets/125534462/2bfec929-9d83-4087-8eef-6c7335f82a0d" alt="WhatsApp Image 2023-12-05 at 21 26 17_897dce8f" width="300" height="500" />

**Comment by Dante291** [comment]
@Cioppolo14 @noman2002 Sir could work on this

**Comment by Dante291** [comment]
@palisadoes @literalEval @Ayush0Chaudhary @noman2002 

I have a suggestion to address the bug. Instead of displaying a demo post in the "My Posts" section of the user's profile, we could implement a solution where only the posts created by the user are shown. If the user hasn't posted anything yet, we can display a text button with a message like "Create Your First Post." This approach provides a cleaner and more personalized user experience, encouraging users to start sharing content.

**Comment by literalEval** [comment]
Sounds interesting to me. Would love to know other's opinions too.

**Comment by ShubhamTiwari55** [comment]
> Sounds interesting to me. Would love to know other's opinions too.

Yes, I agree with @Dante291. Having a functionality of creating posts would be great. It would greatly enhance user experience.

**Comment by Dante291** [comment]
> @palisadoes @literalEval @Ayush0Chaudhary @noman2002
> 
> I have a suggestion to address the bug. Instead of displaying a demo post in the "My Posts" section of the user's profile, we could implement a solution where only the posts created by the user are shown. If the user hasn't posted anything yet, we can display a text button with a message like "Create Your First Post." This approach provides a cleaner and more personalized user experience, encouraging users to start sharing content.

@palisadoes @Cioppolo14 @noman2002 please share your views on this then I will start working on this issue accordingly to that.

**Comment by Dante291** [comment]
https://github.com/PalisadoesFoundation/talawa-api/issues/1589

**Comment by Dante291** [comment]
Working on it
